src/view/com/util/images/ImageLayoutGrid.tsx

```diff
@@ -47,10 +48,10 @@ function ImageLayoutGridInner(props: ImageLayoutGridInnerProps) {
     case 3:
       return (
         <View style={styles.flexRow}>
-          <View style={{flex: 2, aspectRatio: 1}}>
+          <View style={{flex: 2, aspectRatio: isWeb ? 1 : undefined}}>
```

Comment:
Nit: let's move these to `styles` below so that all related hacks are visually close in the code